### PR TITLE
Fix verification tests

### PR DESCRIPTION
**Pull request by jaymalasinha**
Signed-off-by: Jaymala Sinha <jsinha@chef.io>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [x] All tests pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ slack:
 # This publish is triggered by the `built_in:publish_rubygems` artifact_action.
 rubygems:
   - ffi-yajl
+  - ffi-yajl-universla-java
```
Comment (tas50) [inline .expeditor/config.yml]:
```suggestion
  - ffi-yajl-universal-java
```

File: .expeditor/verify.pipeline.yml [hunk 2]
```diff
@@ -1,44 +1,83 @@
...
+    - gem install yajl-ruby json psych
     - bundle exec rake
+  soft_fail:
+      - exit_status: 1
   expeditor:
     executor:
       docker:
-        image: ruby:2.4-stretch
-        privileged: true
+        image: ruby:2.4
```
Comment (tas50) [inline .expeditor/verify.pipeline.yml]: what OS is used in the non-debian images?

File: Gemfile [hunk 3]
```diff
@@ -2,6 +2,11 @@ source "https://rubygems.org"
 
 gemspec name: "ffi-yajl"
 
+group :test do
+  gem "rspec"
```
Comment (tas50) [inline Gemfile]: These should go in development so we can keep our standard groups. Also you can remove the github changelog generator

File: .expeditor/verify.pipeline.yml (hunk 2 above)
Comment (jaymalasinha) [reply .expeditor/verify.pipeline.yml]: it is running `Debian GNU/Linux 10 (buster)`

Comment (tas50) [reply .expeditor/verify.pipeline.yml]: cool

**Pull request merged**